app/index.js

```diff
@@ -23,7 +23,7 @@ module.exports = generators.Base.extend({
                 type: 'input',
                 name: 'name',
                 message: 'Your project name',
-                default: this.appname // Default to current folder name
+                default: 'react-engine-generator' // Default to current folder name
```

Comment:
@ppiekarczyk Thanks for helping to improve the project.  The root of the problem is that the name field in the package.json file should not contain capital letters or spaces.  If we could add a function that will convert the capital letters to lowercase and eliminate the spaces that would be great.  Below is the line where the name field is set in the package.json file.
https://github.com/paypal/generator-react-engine/blob/master/app/index.js#L68
If you would like to make the change please do so, otherwise you can open an issue and I can make the change.